docs/particles.md

```diff
@@ -0,0 +1,73 @@
+# Particles
+
+Parthenon provides a data framework for particle methods that allows for Kokkos
+accelerated parallel dispatch of compute operations. Particle memory is allocated as a
+separate pool for each variable in the particle.
+
+## Swarms
```

Comment:
This does not appear to be updated

Reply:
I think the documentation is still up-to-date (as far as I understand it) because the "Swarms" still contain particles etc. only that the swarms themselves are now stored in a `DataCollection`.
@brryan feel free to update this in the other branch as I'm going to merge this momentarily.